docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/digitalocean-app-platform-2022.md

```diff
@@ -0,0 +1,225 @@
+---
+title: DigitalOcean App Platform Deployment - Strapi Developer Docs
+description: Learn in this guide how to deploy your Strapi application on DigitalOcean App Platform.
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/digitalocean-app-platform-2022.html
+---
+
+# Deploying to DigitalOcean App Platform
+
+The purpose of this guide is to allow users to deploy Strapi applications on the DigitalOcean App Platform. While both Strapi and DigitalOcean App Platform can work with multiple types of databases the focus is on connecting to a postgres development database so users can test their application in a deployed environment. Future additions to the guide will address different database options. This guide does not cover migrating local database content to a production database or connecting front-end applications to the DigitalOcean App Platform.
+
+::: caution
+Strapi maintains deployment guides to assist users in deploying projects. Since there are frequent updates to Strapi and to the hosting provider platforms, the guides are sometimes out of date. If you encounter an issue deploying your project following this guide, please open an issue on GitHub.
+:::
+
+## Prerequisites
+
+Prior to starting the deployment process each user will need:
+
+- a DigitalOcean account ([referral link](https://try.digitalocean.com/strapi/)),
+- a [GitHub account](https://github.com/join),
+- Git version control.
+
+The DigitalOcean App Platform uses version control repositories such as GitHub to deploy applications. In addition to GitHub, GitLab and Docker are also supported. The Strapi referral link for DigitalOcean provides \$100 in credits. Git version control must be used locally, in order to store an application in a remote repository such as GitHub. Follow the details below to setup Git for the project.
+
+::: details - Setting up Git version control and connecting to a remote repository
+
+[Git quick-start guide from GitHub](https://docs.github.com/en/get-started/quickstart/set-up-git)
+
+:::
+
+## Setting up a Strapi project for deployment
+
+Strapi uses [environment configurations](/developer-docs/latest/setup-deployment-guides/configurations/optional/environment.md) to maintain multiple environments inside a single application. This section describes how to setup a production environment in a Strapi application.
+
+1. [Create a new Strapi application using the quickstart flag](/developer-docs/latest/getting-started/quick-start.md) or navigate to the root directory of an existing Strapi application.
```

Comment:
This does not align with all of the existing deployment guides. We can make this change, but it will need to be propagated through the rest of the guides.